src/smt/optimization_solver.h

```diff
@@ -33,63 +33,70 @@ namespace smt {
 /**
  * The optimization result of an optimization objective
  * containing:
- * - whether it's optimal or not
- * - if so, the optimal value, otherwise the value might be empty node or
- *   something suboptimal
+ * - the optimization result: SAT/UNSAT/UNKNOWN
+ * - the optimal value if SAT and bounded,
+ *   or an empty node if SAT and unbounded,
```

Comment:
Please clarify in the comment what is SAT and (un)bounded.

Reply:
this is clarified.